packages/crab-netting/src/CrabNetting.sol

```diff
@@ -490,6 +490,10 @@ contract CrabNetting is Ownable, EIP712 {
      */
     function depositAuction(DepositAuctionParams calldata _p) external onlyOwner {
         _checkOTCPrice(_p.clearingPrice, false);
+        uint256 ethUSDCPrice = IOracle(oracle).getTwap(ethUsdcPool, weth, usdc, auctionTwapPeriod, true);
+        require(
+            (_p.depositsQueued * (1e18 - MAX_OTC_PRICE_TOLERANCE) / ethUSDCPrice) < _p.minEth, "Min Eth Out too low"
```

Comment:
this doesn't take into account decimals of USDC vs ETH

Reply:
it should also probably use `otcPriceTolerance` right not the constant?

Reply:
Thanks for this catch